app/services/rapid_ready_for_decision/views/shared/medications.erb

```diff
@@ -9,13 +9,14 @@
   <i>All VAMC locations using VistA/CAPRI were checked</i> <br />
 </p>
 
-<% unless medications? %>
+<% unless @assessed_data[:medications].any? %>
+  <p><br/></p>
   <h6>No active medications were found in the last year</h6>
 <% end %>
 
 <p> <br /> </p>
 
-<% @medications.each do |medication| %>
+<% @assessed_data[:medications].each do |medication| %>
```

Comment:
If `@assessed_data[:medications]` is nil, this will raise an error. A possible fix:
```suggestion
<% (@assessed_data[:medications] || []).each do |medication| %>
```